runtime/gc_vlhgc/WriteOnceCompactor.cpp

```diff
@@ -1237,7 +1237,9 @@ MM_WriteOnceCompactor::fixupContinuationObject(MM_EnvironmentVLHGC* env, J9Objec
 	fixupMixedObject(env, objectPtr, cache);
 
 	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
-	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr)) {
+	/* In sliding compaction we must fix slots exactly once. Since we will fixup stack slots of mounted Virtual threads later during root fixup,
+	 * we will skip it during this heap fixup pass (hence passing true for scanOnlyUnmounted parameter) */
```

Comment:
- Formatting: See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-comments. The terminating */ of a multi-line comment should appear on a line by itself.
- [Optional] 145 chars per line seems too long. I was told not to exceed 100 chars per line in the past. 
```suggestion
	/* In sliding compaction we must fix slots exactly once. Since we will fixup stack slots of mounted Virtual threads later during root fixup,
	 * we will skip it during this heap fixup pass (hence passing true for scanOnlyUnmounted parameter)
	 */
```